torchinductor/lowering.py

```diff
@@ -490,6 +490,22 @@ def squeeze_(x, dim=None):
     return x
 
 
+@register_lowering(aten.ceil)
+def ceil(x):
+    if is_integer_type(x):
+        return x
+    fn = ops_wrapper(aten.ceil.__name__)
+    return make_pointwise(fn)(x)
+
+
+@register_lowering(aten.floor)
+def floor(x):
+    if is_integer_type(x):
+        return x
+    fn = ops_wrapper(aten.floor.__name__)
```

Comment:
Let's just use `"floor"` directly.  I don't think the `__name__` is making the code more generic, since if it ever changed it would break things.

Reply:
fixed.